core/module.py

```diff
@@ -168,6 +165,10 @@ def carrot2(self, q):
 		search = carrot2.main(self, q)
 		return search
 
+	def duckduckgo(self, q, limit=1):
```

Comment:
@saeeddhqan removing count parameter creates bug. It is still present in module_run and run_crawl.